examples/pytorch/label_propagation/model.py

```diff
@@ -41,16 +41,15 @@ def forward(self, g, labels, mask=None, post_step=lambda y: y.clamp_(0., 1.)):
                 y = torch.zeros_like(labels)
                 y[mask] = labels[mask]
             
-            last = (1 - self.alpha) * y
```

Comment:
Is it possible to replace `LabelPropagation` in this example with the built-in `LabelPropagation` module?